src/pages/index.js

```diff
@@ -0,0 +1,310 @@
+import "../pages/index.css";
+import {
+  enableValidation,
+  settings,
+  disableButton,
+  resetValidation,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c78896b4-8f8f-4aab-b711-572c50f5bd05",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardEl = getCardElement(item);
+      cardList.append(cardEl);
+    });
+
+    avatarElement.src = userInfo.avatar;
```

Comment:
`avatarElement.src = userInfo.avatar;` is not needed because you have it in line 33